FrontEnd/FrontEnd.Web/Components/Pages/Recipes/SingleRecipe.razor

```diff
@@ -0,0 +1,88 @@
+﻿@page "/recipes/{Id:int}"
+
+@using FrontEnd.Web.Services
+@using global::Recipes.Public
+@attribute [StreamRendering]
+@inject IRecipeService RecipeService
+
+<PageTitle>@(_recipe?.Title ?? "Single recipe display")</PageTitle>
+
+@if (_task is not { IsCompleted: true })
+{
+    <FluentSkeleton Height="14rem" Width="100%" Style="margin-bottom: 10px;"/>
+    <FluentSkeleton Height="2.5em" Width="100%" Style="margin-bottom: 10px;"/>
+    <FluentStack Orientation="Orientation.Vertical">
+        <FluentStack>
+            <FluentSkeleton Height="20px" Width="50px"/>
+            <FluentSkeleton Height="20px" Width="50px"/>
+        </FluentStack>
+        <FluentSkeleton Height="0.875em" Width="100%"/>
+        <FluentSkeleton Height="0.875em" Width="100%"/>
+        <FluentSkeleton Height="0.875em" Width="50%"/>
+        <FluentSkeleton Height="1.5em" Width="150px" Style="margin-top: 10px;"/>
+        <FluentSkeleton Height="2em" Width="100%"/>
+        <FluentSkeleton Height="2em" Width="100%"/>
+    </FluentStack>
+}
+else if (_recipe == null)
+{
+    <FluentLabel Typo="Typography.H1" MarginBlock="10px">Recipe not found.</FluentLabel>
+    <FluentLabel Typo="Typography.Body" MarginBlock="10px">The requested recipe doesn't exist.</FluentLabel>
+}
+else
+{
+    <div style="height: 14rem; overflow: hidden; display: flex; justify-content: center; align-items: center;">
+        <img src="https://worldfoodtour.co.uk/wp-content/uploads/2013/06/neptune-placeholder-48.jpg" alt="@_recipe.Title"/>
+    </div>
+    <FluentStack Style="margin-block: 10px;">
+        <FluentLabel Typo="Typography.H1" Style="flex-grow: 1">@_recipe.Title</FluentLabel>
+        <FluentStack Width="auto">
```

Comment:
[`<FluentSpacer/>`](https://www.fluentui-blazor.net/Spacer)? (From source code only, not sure if this makes them align to the right, or `Style="flex-grow: 1"` but the point still stands)

Reply:
Yeah, that works. Addressed.